src/game/default/g_main.c

```diff
@@ -796,21 +799,23 @@ static void G_Frame(void) {
 			return;
 		}
 	}
+		
+	if (!TIMEOUT) {
```

Comment:
This _feels_ dangerous. But I'm assuming you've tested it and nothing goes wonky? Do we effectively suspend `g_level.time` during `TIMEOUT`? Otherwise, plats and things will jump to some new position when `TIMEOUT` expires. I could see lots of things breaking if `g_level.time` is allowed to run during `TIMEOUT`.